### PR TITLE
Update Audit Trail docs [DOC-752]

### PR DESCRIPTION
**Pull request by forstisabella**
### Proposed changes
PM for Consent noted that the Audit Trail docs didn't include a full list of events - I updated them to remove the old list and tentatively [replaced the list of events with product areas](https://twilio.slack.com/archives/C04HNRG3VF1/p1698790770117419), as support for a PAPI-generated list of events the Audit Trail controls for is on the roadmap. Updated the plan note to reflect that [it's no longer an addon for BT customers](https://twilio.slack.com/archives/C04HNRG3VF1/p1699907416367319). Also did some Vale fixes and IA work. 

### Merge timing
ASAP

### CODE REVIEW

File: src/segment-app/iam/audit-trail.md [hunk 1]
```diff
@@ -3,104 +3,54 @@ title: Audit Trail
 plan: audit-trail
 ---
 
-Segment offers an in-app 90 day Audit Trail for Business Tier accounts. If you are a Workspace Owner, you view user and system activity in your workspace settings, in the "Audit Trail" tab under "Admin".
+The Audit Trail allows you can view user and system activity, filter activity for specific actions or actors, and export your data to an event streams source or CSV file.
```
Comment (rchinn-segment) [inline src/segment-app/iam/audit-trail.md:6]:
```suggestion
The Audit Trail allows you to view user and system activity, filter activity for specific actions or actors, and export your data to an event streams source or CSV file. 
```

File: src/segment-app/iam/audit-trail.md [hunk 2]
```diff
@@ -3,104 +3,54 @@ title: Audit Trail
...
+- Journeys
+- Broadcasts 
+- Workspace
+<!--- IG, 11/2023: Add consent to this list when this goes to public beta (q1 '24?)--->
 
-### Destination Filters
-*   Destination Filters Created
-*   Destination Filters Modified
-*   Destination Filters Deleted
+To find a list of all events surfaced in the Audit Trail, open the Audit Trail, click **Filters**, and select the **Events** dropdown.
```
Comment (rchinn-segment) [inline src/segment-app/iam/audit-trail.md:38]:
```suggestion
To view a list of all events Segment surfaces in the Audit Trail, open the Audit Trail, click **Filters**, and select the **Events** dropdown. 
```

File: src/segment-app/iam/audit-trail.md [hunk 3]
```diff
@@ -3,104 +3,54 @@ title: Audit Trail
...
-*   Audience Deleted
-*   Audience CSV Downloaded
-*   Audience Run Failed
-*   Audience Destination Sync Failed
-*   Engage Warehouse Source Created
-*   Engage Warehouse Source Modified
-*   Engage Warehouse Source Deleted
+## Filtering events
 
+The Filters dropdown allows you to refine your search results by filtering by actions or actors to see who made changes on specific resources in the app. Actors include both logged-in users and access tokens.
```
Comment (rchinn-segment) [inline src/segment-app/iam/audit-trail.md:44]:
```suggestion
Use the Filters dropdown to refine your search results and filter by actions or actors to see who made changes on specific resources in the app. Actors include both logged-in users and access tokens. 
```

File: src/segment-app/iam/audit-trail.md [hunk 4]
```diff
@@ -3,104 +3,54 @@ title: Audit Trail
...
 
+The Filters dropdown allows you to refine your search results by filtering by actions or actors to see who made changes on specific resources in the app. Actors include both logged-in users and access tokens. 
 
 ## Audit forwarding
 
-You can forward events in your workspace to supported sources that you have configured in the workspace.
+You can forward events in your workspace to an [event streams source](/docs/connections/sources/#event-streams-sources) to set up real-time alerts and quickly revert changes (like a user unintentionally disabling a warehouse) that could cause unwanted downstream effects.
 
-To forward Audit Trail events:
+> info "Segment recommends creating a dedicated source for audit trail events"
```
Comment (rchinn-segment) [inline src/segment-app/iam/audit-trail.md:50]:
```suggestion
> info "Segment recommends creating a dedicated source for Audit Trail events"
```

Comment (rchinn-segment) [reply src/segment-app/iam/audit-trail.md:50]: Noticed that Audit Trail is capitalized throughout most of the doc, so just flagging for your review.

**Review by rchinn-segment — APPROVED** [review]
Looks good! Just a few minor suggestions for your review.

File: src/segment-app/iam/audit-trail.md (hunk 4 above)
Comment (forstisabella) [reply src/segment-app/iam/audit-trail.md:50]: yeah good catch!

**Pull request merged**